python/cudf_polars/cudf_polars/experimental/rapidsmpf/core.py

```diff
@@ -102,7 +104,10 @@ def evaluate_logical_plan(
                 mr, limit=int(total_memory * single_spill_device)
             )
         }
-    br = BufferResource(mr, memory_available=memory_available)
+
+    # Temporarily testing with a stream pool of size 1
+    stream_pool = CudaStreamPool(pool_size=1, flags=CudaStreamFlags.NON_BLOCKING)
```

Comment:
Should we temporarily allow the user to control this with an environment variable (to simplify testing)?